ironfish/src/primitives/blockheader.ts

```diff
@@ -150,6 +150,8 @@ export class BlockHeader {
    */
   public readonly graffiti: Buffer
 
+  public readonly stateRoot?: Buffer
```

Comment:
The actual name of this is not super-important, since we could theoretically rename it without a fork, but I feel like using `commitment` instead of `root` might better match the other fields on the block header

Reply:
That sounds good to me, I'm not afraid of `commitment`